src/amber/cli/commands/migrate.cr

```diff
@@ -20,25 +20,26 @@ module Amber::CLI
       end
 
       def run
+        migrations_path, migrations_table_suffix = Micrate::Cli.parse_command_arguments
```

Comment:
Ditto.

Reply:
I just fixed it to work with latest version of micrate shard see: https://github.com/juanedi/micrate/commit/af301fcec0c42ee5872fe0d783fbf91a762a11b9

Reply:
I reverted these changes, please check this PR again 😄